sql/src/SqlPrivacy.sk

```diff
@@ -549,13 +564,13 @@ fun checkUserCanWriteRow(
     };
     hasAdminPrivileges = {
       groupIndex = skdbGroups.getIndex(context);
-      adminID = groupIndex.get(skdbGroups.makeKey(groupID)) match {
-      | Some(r) -> r.getString(adminIDColName)
-      | None() -> throw SqlError(0, "malformed predefined table skdb_groups")
+      groupIndex.get(skdbGroups.makeKey(groupID)) match {
+      | Some(r) ->
+        adminID = r.getString(adminIDColName);
+        (row.repeat == 0 && access.canDelete(context, adminID)) ||
+          access.canInsert(context, adminID)
+      | None() -> return AROK()
```

Comment:
Hmm, this should never happen if we've properly initialized privacy tables, but it makes me a little uncomfortable for the default behaviour to be permissive instead of erroring out if we reach this case.

Reply:
It happens now because I made group creation atomic. I had to break the cyclic dependencies. So I reasoned it's ok to create permissions for groups that don't exist yet.

Reply:
(Because anyone can create a group and you obviously won't create a group that already has permissions you don't want and can't change.)